### PR TITLE
Clarify that the API request size limit is 10MB, not 5MB.

### PR DESCRIPTION
**Pull request by qingling128**

### CODE REVIEW

File: templates/etc/td-agent/td-agent.conf [hunk 1]
```diff
@@ -38,7 +38,8 @@
   buffer_type file
   buffer_path /var/log/google-fluentd/buffers
   # Set the chunk limit conservatively to avoid exceeding the recommended
-  # chunk size of 5MB per write request.
+  # chunk size of 10MB per write request. The API request size can be a few
+  # times biggers than the raw log size.
```
Comment (igorpeshansky) [inline templates/etc/td-agent/td-agent.conf:42]: Typo: biggers.

File: templates/etc/td-agent/td-agent.conf.tmpl [hunk 2]
```diff
@@ -38,7 +38,8 @@
   buffer_type file
   buffer_path /var/log/google-fluentd/buffers
   # Set the chunk limit conservatively to avoid exceeding the recommended
-  # chunk size of 5MB per write request.
+  # chunk size of 10MB per write request. The API request size can be a few
+  # times biggers than the raw log size.
```
Comment (igorpeshansky) [inline templates/etc/td-agent/td-agent.conf.tmpl:42]: Typo: biggers.

**Review by igorpeshansky — CHANGES_REQUESTED** [review]

**Review by qingling128 — COMMENTED** [review]
Nice catch! Fixed the typo.

File: templates/etc/td-agent/td-agent.conf [hunk 3]
```diff
@@ -38,7 +38,8 @@
   buffer_type file
   buffer_path /var/log/google-fluentd/buffers
   # Set the chunk limit conservatively to avoid exceeding the recommended
-  # chunk size of 5MB per write request.
```
Comment (igorpeshansky) [inline templates/etc/td-agent/td-agent.conf:41]: Do we need the same change in [windows-installer/fluent-template.conf](https://github.com/GoogleCloudPlatform/google-fluentd/blob/master/windows-installer/fluent-template.conf#L33-L34)?

**Review by igorpeshansky — CHANGES_REQUESTED** [review]
One more place for this change.

File: templates/etc/td-agent/td-agent.conf (hunk 3 above)
Comment (qingling128) [reply templates/etc/td-agent/td-agent.conf:41]: Updated.

**Review by qingling128 — COMMENTED** [review]
PTAL

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Pull request merged**